cldf/cldf-metadata.json

```diff
@@ -11,14 +11,14 @@
     "dc:license": "https://creativecommons.org/licenses/by/3.0/",
     "dc:related": null,
     "dc:source": "sources.bib",
-    "dc:title": "The World Loanword Database",
+    "dc:title": "CLDF dataset derived from Haspelmath & Tadmor \"WOLD\" (2009)",
```

Comment:
Yes, please check with my instructions. It is AUTHOR'**s**, and WOLD is not a good title, as it is too short, and it is more explicit if one spells it out.

Reply:
Fixed with 9325f52